docfx.json

```diff
@@ -192,20 +192,20 @@
         "docs/core/whats-new/**/**.md": "Thraka",
         "docs/csharp/**/**.md": "billwagner",
         "docs/csharp/**/**.yml": "billwagner",
-        "docs/framework/**/**.md": "mairaw",
+        "docs/framework/**/**.md": "dotnet/docs",
         "docs/framework/app-domains/**/**.md": "gewarren",
         "docs/framework/configure-apps/file-schema/network/**/**.md": "karelz",
         "docs/framework/configure-apps/file-schema/wcf/**/**.md": "HongGit",
         "docs/framework/configure-apps/file-schema/winforms/**/**.md": "Thraka",
         "docs/framework/data/**/**.md": "stevestein",
         "docs/framework/deployment/**/**.md": "Thraka",
-        "docs/framework/get-started/**/**.md": "mairaw",
+        "docs/framework/get-started/**/**.md": "gewarren",
         "docs/framework/install/**/**.md": "Thraka",
         "docs/framework/migration-guide/**/**.md": "gewarren",
-        "docs/framework/misc/**/**.md": "mairaw",
-        "docs/framework/net-native/**/**.md": "mairaw",
+        "docs/framework/misc/**/**.md": "gewarren",
+        "docs/framework/net-native/**/**.md": "gewarren",
         "docs/framework/network-programming/**/**.md": "karelz",
-        "docs/framework/performance/**/**.md": "mairaw",
+        "docs/framework/performance/**/**.md": "billwagner",
```

Comment:
I used @BillWagner since he owns the threading docs which are also about perf in a way, but let me know if it should be someone else or just the dotnet/docs team.